cli/command/cli.go

```diff
@@ -180,6 +180,10 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 }
 
 func isEnabled(value string) (bool, error) {
+	// Environment variable always overrides configuration
+	if os.Getenv("DOCKER_CLI_EXPERIMENTAL") != "" {
```

Comment:
Can you update the documentation, and add this env-var to the list in https://github.com/docker/cli/blob/master/docs/reference/commandline/cli.md#environment-variables ?

Reply:
So this means that `DOCKER_CLI_EXPERIMENTAL=false docker manifest` will be valid? Maybe we should match the configuration file values instead of just checking the existence of the environment variable?

Reply:
This does not seem very consistent at the moment, for instance there is
```
func contentTrustEnabled() bool {
	if e := os.Getenv("DOCKER_CONTENT_TRUST"); e != "" {
		if t, err := strconv.ParseBool(e); t || err != nil {
			// treat any other value as true
			return true
		}
	}
	return false
}
```
but
```
func hide(cmd *cobra.Command) *cobra.Command {
	// If the environment variable with name "DOCKER_HIDE_LEGACY_COMMANDS" is not empty,
	// these legacy commands (such as `docker ps`, `docker exec`, etc)
	// will not be shown in output console.
	if os.Getenv("DOCKER_HIDE_LEGACY_COMMANDS") == "" {
		return cmd
	}
…
```

Reply:
Oh right...

Reply:
Yeah for it to be uniform with what the value in the config is, it would need to be either `enabled` or `disabled`. Should I change it to that?

Reply:
Here's what the diff would look like for that: